### DISCUSSION TITLE
Confuse on expanding sub relationship.

### DISCUSSION THREAD

**Discussion by indrazm** [pocketbase/pocketbase#3528] [Q&A]
Hey there i have a user table that have relationship to batch table.

```js
async function getLeaderboard() {
  const leaderboard = await pocketbase.collection('leaderboard').getFullList({
    expand: `user`,
  });
  return leaderboard;
}
```

i want to do below, but i can't.

```js
async function getLeaderboard() {
  const leaderboard = await pocketbase.collection('leaderboard').getFullList({
    expand: `user,user.batch`,
  });
  return leaderboard;
}
```

Is there any way to do it ?
Many Thanks!

**Comment by ganigeorgiev** [comment]
`expand: 'yourRelField.yourSubRelField'` should work.

Note that the client request must have View API rule access for the collections that are referenced by `yourRelField` and `yourSubRelField`.

If that doesn't help, please export your collections configuration as JSON (Admin UI > Settings > Export collections).
